packages/cardpay-cli/safe/view.ts

```diff
@@ -16,12 +16,12 @@ export default {
       .option('network', NETWORK_OPTION_LAYER_2);
   },
   async handler(args: Arguments) {
-    let { network, mnemonic, safeAddress } = args as unknown as {
+    let { network, safeAddress } = args as unknown as {
       network: string;
-      mnemonic?: string;
```

Comment:
@habdelra With the addition of `getWeb3Opts` im able to remove mnemonic to be passed int every single subcommand.